admin_guide/limits.adoc

```diff
@@ -87,7 +87,36 @@ specified.
 <13> The default amount of memory that a container will request to use if not specified.
 <14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
 ====
-// end::admin_limits_sample_definition[]
+
+.OpenShift Limit Range Object Definition
+====
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "openshift-resource-limits"
+spec:
+  limits:
+    -
```

Comment:
> This is actually a valid yaml. The dash preceding an array item which is an object cannot be on the same line as the object itself. See another object definitions above.

I'm just saying it reads better if it's in the same line. I'm a lazy dev who doesn't bother looking into the generated version locally :wink: